public/app/features/dimensions/scaler.ts

```diff
@@ -0,0 +1,60 @@
+import { DataFrame, Field } from '@grafana/data';
+import { DimensionSupplier, ScalerDimensionConfig, ScalerDimensionMode } from './types';
+import { findField, getLastNotNullFieldValue } from './utils';
+
+//---------------------------------------------------------
+// Scaler dimension
```

Comment:
@nikki-kiga i think this is supposed to be Scalar?

https://en.wikipedia.org/wiki/Scalar

Reply:
Agree onn changing naming at this stage (hopefully shouldn't be too bad with VSCode refactoring tools 🤞 )